doc/releases/release-notes-3.3.rst

```diff
@@ -1088,6 +1101,12 @@ Libraries / Subsystems
   * ``CONFIG_HAS_EXTERNAL_CACHE`` has been renamed to
     :kconfig:option:`CONFIG_EXTERNAL_CACHE`
 
+* DSP
+
+  * Introduced DSP (digital signal processing) subsystem with CMSIS-DSP as the
+    default backend.
+  * CMSIS-DSP now supports all architectures supported by Zephyr.
```

Comment:
Note needed on renaming to zdsp?

Reply:
No, CMSIS-DSP has not been renamed to ZDSP. It is just the default backend for the ZDSP (DSP subsystem).